style.css

```diff
@@ -295,3 +350,54 @@ footer {
 .element-display-remove {
   display: none !important;
 }
+
+@media screen and (max-width: 970px) {
+  #tasksNav {
+    justify-content: space-between;
+  }
+
+  .sign-in-btn a {
+    position: static;
+  }
+
+  .sign-in-btn span {
+    display: none;
+  }
+  .logo {
+    order: 2;
+  }
+
+  .nav-links .hamburger {
+    order: 1;
+    display: inline-block;
+    margin: auto;
+    cursor: pointer;
+  }
+
+  .nav-links .hamburger .line {
+    width: 25px;
+    margin: 5px;
+    border: 2px solid var(--white-color);
+  }
+
+  .nav-links .links {
+    display: none;
+  }
+
+  .nav-links .active {
+    position: absolute;
+    top: 80px;
+    left: 0;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    background-color: var(--white-color);
+    width: 100%;
+    padding: 10px;
+  }
+  
+  .nav-links .links a {
+    color: var(--blue-color);
+  }
+
```

Comment:
When working with media query, it's better to have as little things inside media query as possible.

From the functionality I see, you would need to only use - 

display: none/block
&
justify-content/align-items: center

Reply:
Ok. Shall I make the changes?

Reply:
Yes please